packages/office-ui-fabric-react/src/components/Toggle/Toggle.styles.ts

```diff
@@ -79,28 +79,28 @@ export const getStyles = memoizeFunction((
     },
 
     thumb: {
-      width: '.5em',
-      height: '.5em',
       borderRadius: '.5em',
       position: 'absolute',
       top: '.2em',
       transition: 'all 0.1s ease',
-      backgroundColor: thumbBackground,
+      borderColor: thumbBackground,
```

Comment:
@phkuo, I guess that is fixable from user end when they realize that this is using borders so overriding shouldn't be an issue. The bigger problem we are fighting here is the firefox HC - it doesn't recognize the backgrounds and the only resort is to use borders. We are doing same for other components like Checkbox, Pivots etc. I am in for exploring better alternative but it should not a blocker for this PRs :)